openai/openai.go

```diff
@@ -27,7 +28,7 @@ type ErrorResponse struct {
 
 type Message struct {
 	Role    string `json:"role"`
-	Content string `json:"content"`
+	Content any    `json:"content"`
```

Comment:
Let's add some tests for this one, especially since we're now accepting `any` for this field